src/auth/negotiate/wrapper/negotiate_wrapper.cc

```diff
@@ -224,6 +224,13 @@ processingLoop(FILE *FDKIN, FILE *FDKOUT, FILE *FDNIN, FILE *FDNOUT)
                         LogTime(), PROGRAM);
                 return 0;
             }
+
+            if (!memchr(tbuff, '\n', sizeof(tbuff) - 1)) {
+                fprintf(stderr, "%s| %s: Oversized NTLM helper response\n",
+                        LogTime(), PROGRAM);
```

Comment:
This PR code mimics a similar memchr() check and "Oversized" error message located earlier in this code, where fgets(stdin) is called.